src/environments/environment.ts

```diff
@@ -13,7 +13,7 @@ export const environment = {
   production: false,
   localApiRoot: '/assets',
   remoteApiRoot: 'http://localhost:3000/api',
-  useMockApi: true,
+  useMockApi: false,
```

Comment:
Just some rando thoughts here: this introduces a cross-repo dependency (since the api code is in a different repo). I don't think we want to monorepo for these reference apps because it could introduce that blurs the focus of each individual reference app, so it's worth considering how any updates to the remote service are managed by it clients.

Reply:
I'm going to create another PR to refactor the ProductService.  This can internalize the mock versus remote APIs better.